let responce = fetch("url");


fetch(url)
    .then((responce) =>{
        //handle the responce;
    })
    .catch((error)=>{
        //handles the error;
    });